rundeckapp/grails-app/domain/rundeck/Option.groovy

```diff
@@ -205,12 +208,25 @@ public class Option implements Comparable{
                 map.multivalueAllSelected = true
             }
         }
+        if(inputType){
+            map.inputType = inputType
+        }
         if(secureInput){
             map.secure=secureInput
         }
+        if(enforced == true) {
+            map.put("enforcedType", "enforced")
+            map.enforced = true
+        }
+        if(enforced == "regex"){
```

Comment:
this doesn't make sense, the `enforced` field is a Boolean, so can't be compared to a String